matrix_sdk/src/error.rs

```diff
@@ -106,6 +107,10 @@ pub enum Error {
     /// An error occured in the state store.
     #[error(transparent)]
     StateStore(#[from] StoreError),
+
+    /// An error encountered when trying to parse an invalid identifier string.
```

Comment:
```suggestion
    /// An error encountered when trying to parse an identifier string.
```

Sounds a bit weird as-is.

Reply:
Maybe remove `string` too.